project/make/test-integration

```diff
@@ -11,6 +11,7 @@ INIT=$DEST/../dynbinary/dockerinit-$VERSION
 export TEST_DOCKERINIT_PATH="$INIT"
 
 bundle_test_integration() {
+	export PATH=$PATH:/go/bin
```

Comment:
These two are added due to the issue that registry tests are added under /go. Technically those tests I believe should not be part of the Docker build?